transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModelTest.java

```diff
@@ -358,7 +374,7 @@ public Object answer(InvocationOnMock invocation) {
             }
         }).when(mTransferRepository).createTransfer(any(Transfer.class),
                 any(TransferRepository.CreateTransferCallback.class));
-
+        viewModel.init();
```

Comment:
don't you want to verify which load is being called?

Reply:
FYI: Discussed. Implemented in some other method.